src/main/scala/datastax/astra/migrate/DiffData.scala

```diff
@@ -17,17 +17,17 @@ object DiffData extends AbstractJob {
   private def diffTable(sourceConnection: CassandraConnector, destinationConnection: CassandraConnector) = {
     val partitions = SplitPartitions.getRandomSubPartitions(splitSize, minPartition, maxPartition, Integer.parseInt(coveragePercent))
     logger.info("PARAM Calculated -- Total Partitions: " + partitions.size())
-    val parts = sc.parallelize(partitions.toSeq, partitions.size);
+    val parts = sContext.parallelize(partitions.toSeq, partitions.size);
     logger.info("Spark parallelize created : " + parts.count() + " parts!");
 
     parts.foreach(part => {
       sourceConnection.withSessionDo(sourceSession =>
         destinationConnection.withSessionDo(destinationSession =>
-          DiffJobSession.getInstance(sourceSession, destinationSession, sc.getConf)
+          DiffJobSession.getInstance(sourceSession, destinationSession, sc)
```

Comment:
Are these (`sourceSession`, `destinationSession`) the ones that you mentioned "will be updated in a future PR"?

Reply:
Yes, there are many places in code where we use terms like `source`, `destination` & `Astra`. All those need to be fixed as well (in a future PR) but will make this PR too big. This one is focused on getting only external facing config file corrected i.e. support `origin/target`, but also provide backward compatibility to `source/destination` (as we dont want current customers to change config). All the code changes are to support this updated config.